shuffle/move/examples/main/sources/Message.move

```diff
@@ -14,13 +15,18 @@ module Sender::Message {
     /// There is no message present
     const ENO_MESSAGE: u64 = 0;
 
-    public(script) fun set_message(account: signer, message: vector<u8>)
+    public fun get_message(a: address): vector<u8> acquires MessageHolder {
+        assert!(exists<MessageHolder>(a), Errors::not_published(ENO_MESSAGE));
+        *&borrow_global<MessageHolder>(a).message
+    }
+
+    public fun set_message_internal(account: &signer, message: vector<u8>)
```

Comment:
I was presented with a catch 22:
1. From the `MessageTests` below, I cannot invoke `set_message(account: signer, ...)` because `signer` cannot be copied. So I try to add `&` to borrow, but then
2. I cannot invoke `set_message(account: &signer, ...)` from a script function because references cannot be called offchain.

As a result, I had to split them out, and have the script function value `set_message(account: signer, ...)` proxy to this `set_message_internal(account: &signer, ...)` while `MessageTests` can now `set_message_internal(account: &signer, ...)` and not need to copy.

Is this a common problem and is there a better/accepted pattern or naming convention to address it?

cc @emmazzz @tzakian

Reply:
For this PR, you can simply call `get_account` multiple times -- this will return the same address every time it's called since `create_signers_for_testing(N)` creates `N` unique signers in a deterministic (and non-stateful) fashion.

Medium/long-term, Emma (and I believe others from the hackathon) had a similar ask, and it's not hard to add an additional test-only native function the `UnitTest` module that would allow you to copy a signer. I can try and have that up in the next day or so (shouldn't be hard to do at all really).